modules/sdk/gradle-plugins-maven-plugin-builder/src/main/java/com/liferay/gradle/plugins/maven/plugin/builder/BuildPluginDescriptorTask.java

```diff
@@ -325,6 +335,36 @@ protected void appendDependencyExclusionElement(
 		XMLUtil.appendElement(doc, exclusionElement, "groupId", groupId);
 	}
 
+	protected void appendPomRepositoryElements(
+		Document doc, Element repositoryElement, String configurationName,
+		String scope) {
+
+		Project project = getProject();
+
+		ConfigurationContainer configurationContainer =
+			project.getConfigurations();
+
+		Configuration configuration = configurationContainer.findByName(
+			configurationName);
+
+		if (configuration == null) {
+			return;
+		}
+
+		Set<Dependency> dependencies = configuration.getDependencies();
+
+		for (Dependency entry : dependencies) {
+			Element dependencyElement = doc.createElement("repository");
```

Comment:
Be careful about variable names: you're creating an XML element called `repository`, so why don't you call the variable `repositoryElement`?